src/app/analytics/analytics.factory.js

```diff
@@ -1,40 +1,39 @@
-import './analytics.module';
-
-import get from 'lodash/get';
-import find from 'lodash/find';
-import sha3 from 'crypto-js/sha3';
-import merge from 'lodash/merge';
-import isEmpty from 'lodash/isEmpty';
+import './analytics.module'
+import angular from 'angular'
+import get from 'lodash/get'
+import find from 'lodash/find'
+import sha3 from 'crypto-js/sha3'
+import merge from 'lodash/merge'
+import isEmpty from 'lodash/isEmpty'
+/* global localStorage */
```

Comment:
Shouldn't this be part of the standard config? Or you could do `$window.localStorage`...

Reply:
I added it here instead of global config so that we'd get a warning if we used it in other places.